src/handler.c

```diff
@@ -671,17 +671,25 @@ struct char_data *get_char_num(mob_rnum nr)
 /* put an object in a room */
 void obj_to_room(struct obj_data *object, room_rnum room)
 {
-  if (!object || room == NOWHERE || room > top_of_world)
+  if (!object || room == NOWHERE || room > top_of_world){
     log("SYSERR: Illegal value(s) passed to obj_to_room. (Room #%d/%d, obj %p)",
 	room, top_of_world, (void *)object);
-  else {
-    object->next_content = world[room].contents;
-    world[room].contents = object;
-    IN_ROOM(object) = room;
-    object->carried_by = NULL;
-    if (ROOM_FLAGGED(room, ROOM_HOUSE))
-      SET_BIT_AR(ROOM_FLAGS(room), ROOM_HOUSE_CRASH);
   }
+	else {
```

Comment:
A little bit "special" indentation in this file.

Reply:
Requested corrections have been made.